src/test/java/com/google/cloud/tools/appengine/cloudsdk/StringBuilderProcessOutputLineListenerTest.java

```diff
@@ -14,19 +14,18 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.appengine.cloudsdk.internal.process;
+package com.google.cloud.tools.appengine.cloudsdk;
```

Comment:
Shouldn't this be `c.g.c.t.a.cloudsdk.process`? That'll also remove the `import` below.